tabular/src/autogluon/tabular/models/rf/rf_model.py

```diff
@@ -27,6 +27,12 @@ class RFModel(AbstractModel):
     """
     Random Forest model (scikit-learn): https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html
     """
+    ag_key = "RF"
+    ag_name = "RandomForest"
+    ag_priority = 80
+    ag_priority_by_problem_type = {
+        SOFTCLASS: 80
+    }
```

Comment:
Is this a typo?

Reply:
Not a typo, but it ends up being redundant. This logic was here to perfectly replicate the old code, but since 80 is also the default, I've removed this now.